pkg/appgateserver/server.go

```diff
@@ -300,13 +300,13 @@ func (app *appGateServer) ServePprof(ctx context.Context, addr string) error {
 	// If no error, start the server in a new goroutine
 	go func() {
 		app.logger.Info().Str("endpoint", addr).Msg("starting a pprof endpoint")
-		server.ListenAndServe()
+		_ = server.ListenAndServe()
```

Comment:
Should we do something here if there's an error?

Reply:
Let's add an error log here. That way, the user/operator can know why the endpoint is not available if something goes wrong (such as a port already in use).